pkg/utils/string.go

```diff
@@ -0,0 +1,10 @@
+package utils
+
+func StringInSlice(a string, list []string) bool {
```

Comment:
if you you moved to go 1.21, it has a contains method:
https://pkg.go.dev/slices#Contains